controllers/apps/apimanager_controller.go

```diff
@@ -64,6 +64,7 @@ var _ reconcile.Reconciler = &APIManagerReconciler{}
 // +kubebuilder:rbac:groups=policy,namespace=placeholder,resources=poddisruptionbudgets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=monitoring.coreos.com,namespace=placeholder,resources=podmonitors;servicemonitors;prometheusrules,verbs=get;list;watch;create;update;delete
 // +kubebuilder:rbac:groups=integreatly.org,namespace=placeholder,resources=grafanadashboards,verbs=get;list;watch;create;update;delete
+// +kubebuilder:rbac:groups=config.openshift.io,namespace=placeholder,resources=clusterversions,verbs=get;list
```

Comment:
I haven't been able to verify the fix. Followed the verification steps and the error continues showing up. Perhaps the permission should be added in the ClusterRole? 

```suggestion
// +kubebuilder:rbac:groups=config.openshift.io,resources=clusterversions,verbs=get;list
```

Reply:
If I am not wrong, the `namespace=placeholder` is used for the namespaced deployment of the operator. Otherwise, what is it's purpose?

If  that would be deleted, why not from the other rbac deficinitions? I think we should keep it. 

If you deployed the operator in its own deployment and pod and still get the errors, we should do pair reviewing to see what you did differently

Reply:
The reason to remove the `namespace` part of the marker is that when the namespace is not specified, Kubebuilder will set this permission in the `ClusterRole` (instead of the `Role`)

I deployed the operator with OLM, generated the manifests from this PR and even with the permission set in the role, the error was still showing up. I manually added the permission to the ClusterRole and it seemed to work shortly after

Reply:
Oh, I see. The clusterversion is not namespaced resource, you need cluster role. 

When I run the verification steps, I might have deployed the operator cluster wide, which effectively turns permissions in clusterroles and the permission issue was gone. 

yes, indeed... we need to remove that, good catch.